GramAddict/core/filter.py

```diff
@@ -251,6 +273,13 @@ def can_follow_private_or_empty(self):
             field_follow_private_or_empty
         )
 
+    @staticmethod
+    def _get_follow_button_text(device):
+        profileView = ProfileView(device)
+        follower = profileView.getFollowButton()
+
+        return follower[1]
```

Comment:
Please, check my suggestion in views.py. If we implement that, there is no need to create this method. We would improve code readability since `return follower[1]` doesn't say much.